src/main/resources/archetype-resources/deployment/k8s/service.yaml

```diff
@@ -3,15 +3,16 @@ apiVersion: v1
 kind: Service
 
 metadata:
-  name: ${projectName}
+  name: ${rootArtifactId}-silo
```

Comment:
Looks like you are reverting the changes from https://github.com/rebuy-de/rebuy-silo-archetype/pull/171 here.